### PR TITLE
feat(dashboard/admin): implemented a waive fines button on admin single user view

### LINKED ISSUE #123: Fine waive functionality
In the frontend there should be a way of waiving a users fines. This can simply be done with a button calling the `/users/{id}/fines/waive` endpoint on the backend. 

- [ ] Make button that waives a users' fines
- [ ] Make it so the mutation table correctly shows this mutation (and not unkown transaction).

### PR DESCRIPTION
**Pull request by CodeNamedRobin**
# Description
Created a way for an admin, or BAC PM, to waive the fines of a single user. 
## Related issues/external references
Closes #123

## Types of changes
- New feature _(non-breaking change which adds functionality)_

### CODE REVIEW

File: apps/dashboard/src/locales/en.json [hunk 1]
```diff
@@ -266,7 +269,10 @@
     "payoutDenied": "Successfully denied payout.",
     "pointOfSaleUpdated": "Successfully updated point of sale.",
     "pointOfSaleDeleted": "Successfully deleted point of sale.",
-    "pointOfSaleCreated": "Successfully created point of sale."
+    "pointOfSaleCreated": "Successfully created point of sale.",
+    "waiveFinesSuccess": "Successfully waived fines.",
+    "waiveFinesRejected": "Canceled waiving fines.",
```
Comment (BHenkemans) [inline apps/dashboard/src/locales/en.json:274]: I would argue that these are not part of a product container operation.

File: apps/dashboard/src/locales/nl.json [hunk 2]
```diff
@@ -267,7 +270,10 @@
     "invoiceUpdated": "Factuur {reference} is succesvol geüpdatet.",
     "pointOfSaleUpdated": "Point of sale is succesvol geüpdatet.",
     "pointOfSaleDeleted": "Point of sale is successvol verwijdered.",
-    "pointOfSaleCreated": "Point of sale is successvol aangemaakt."
+    "pointOfSaleCreated": "Point of sale is successvol aangemaakt.",
+    "waiveFinesSuccess": "Boetes zijn succesvol kwijtgescholden.",
+    "waiveFinesRejected": "Boetes zijn succesvol afgewezen.",
```
Comment (BHenkemans) [inline apps/dashboard/src/locales/nl.json:275]: See previous comment

**Review by BHenkemans — CHANGES_REQUESTED** [review]
Some translations are not properly sorted imo

File: apps/dashboard/src/locales/en.json (hunk 1 above)
Comment (CodeNamedRobin) [reply apps/dashboard/src/locales/en.json:274]: Will fix

**Review by SuperVK — APPROVED** [review]
With Bartjan's changes it's good!

**Review by BHenkemans — APPROVED** [review]
Great!

**Pull request merged**